doc/INSTALLING.md

```diff
@@ -1,22 +1,27 @@
 ### Installing for production
+
 The project is on npm at https://www.npmjs.com/package/clappr
 
 `npm install clappr --save-dev`
 
 You should specify the base url for where the assets are located using the `baseUrl` option:
+
 ```javascript
-  var player = new Clappr.Player({
-  	source: "http://your.video/here.mp4",
-	baseUrl: "http://example.com/assets/clappr"
-  });
+var player = new Clappr.Player({
+  source: "http://your.video/here.mp4",
+  baseUrl: "http://example.com/assets/clappr"
+});
 ```
+
```

Comment:
Try to avoid adding extra empty lines before or after code blocks. It only increases the file length, but with no big benefit, since the block start or end line contains only the ``` markers (and the language, optionally).